catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -519,7 +519,7 @@ public Response getDocument(@PathParam("sourceid") String sourceid, @PathParam("
                 LOGGER.warn(exceptionMessage, e);
                 throw new ServerErrorException(exceptionMessage, Status.INTERNAL_SERVER_ERROR);
             } catch (CatalogTransformerException e) {
-                String exceptionMessage = "Unable to transform Metacard.  Try different transformer: ";
+                String exceptionMessage = "Unable to transform Metacard : \n" + e.getMessage();
```

Comment:
I would then say that you should throw up a different runtime exception for the data limit, just like we have the Internal server exception. Then you don't have to wonder whether or not someone is exposing things that shouldn't be exposed through this broader exception type.